build.fsx

```diff
@@ -34,6 +34,7 @@ open Fake.DotNet.Cli
 open Fake.Testing.Common
 open Fake.DotNet.Testing.MSpec
 open Fake.DotNet.Testing.XUnit2
+open Fake.DotNet.Testing.MSTest
```

Comment:
We have a small bootstrapping problem with this ;). I think we can remove those lines for now.